packages/main/src/themes/base/StepInput-parameters.css

```diff
@@ -20,4 +20,5 @@
 	--_ui5_step_input_border_color_hover: var(--sapField_Hover_Background);
 	--_ui5_step_input_border_hover: 1px solid var(--sapField_Hover_BorderColor);
 	--_ui5_input_input_background_color: var(--sapField_InvalidBackground);
+	--_ui5_step_input_min_width: 7.25rem;
```

Comment:
Shouldn't it be 6.25rem if we are calculating the sum of the two buttons plus the paddings?
In cozy and compact each of the two buttons has 40px width given the padding. There is also additional padding
which is 10px on cozy from each side and 8px for compact. The calculation will be (40 + 40 + 10 + 10) % 16 for cozy and (40 + 40 + 8 + 8) % 16 for compact right?

Reply:
We calculate 2 Buttons + paddings + input. For cosy is 2.25 + 2.25 + 2.75 = 7.25rem.

Reply:
Yes that adds up.